### PR TITLE
Resolve #754: refactor View.dp() as Resources.pxToDp

### LINKED ISSUE #754: View.dp has nothing to do with View
We have a function `View.dp()` that converts px to dp:
https://github.com/mozilla-mobile/android-components/blob/d055cc58566737697b360bd08728b80a7bffa91c/components/support/ktx/src/main/java/mozilla/components/support/ktx/android/view/View.kt#L32-L33

However, this has nothing to do with a View, which makes it hard to find and clutters View with confusing methods. I would propose one of:
```kt
fun Int.pxToDp(context: Context)
fun Resources.pxToDp(pixels: Int)
```

### PR DESCRIPTION
**Pull request by SubhrajyotiSen**

### CODE REVIEW

File: components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package mozilla.components.support.ktx.android.content.res
+
+import android.util.DisplayMetrics
+import org.junit.Assert
+import org.junit.Test
+import org.robolectric.RuntimeEnvironment
+
+@Test
```
Comment (csadilek) [inline components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt]: Similar to `ViewTest` can we rename this file to `ResourcesTest` and create a test class for it?

File: components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+fun `dp returns same value as manual conversion`() {
```
Comment (csadilek) [inline components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt]: Similar to ViewTest can we rename this file to ResourcesTest and create a test class for it?

**Comment by csadilek** [comment]
Thanks @SubhrajyotiSen. @mcomella when you get a chance, can you take a look at this as well?

File: components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt (hunk 2 above)
Comment (SubhrajyotiSen) [reply components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt]: I will take a look

**Comment by SubhrajyotiSen** [comment]
I will fix the ktlint failure when I'm back at my computer later today

File: components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+import android.util.DisplayMetrics
+import org.junit.Assert
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment
+
+@RunWith(RobolectricTestRunner::class)
+class ResourcesTest {
```
Comment (csadilek) [inline components/support/ktx/src/test/java/mozilla/components/support/ktx/android/content/res/Resources.kt]: @SubhrajyotiSen I think the filename should be ResourcesTest.kt as well for consistency. Thanks!

**Pull request merged**